sdk/communication/communication-chat/test/public/chatThreadClient.mocked.spec.ts

```diff
@@ -13,9 +13,9 @@ import {
   SendMessageOptions,
   UpdateMessageOptions,
   AddParticipantsRequest
-} from "../src";
-import * as RestModel from "../src/generated/src/models";
-import { apiVersion } from "../src/generated/src/models/parameters";
+} from "../../src";
+import * as RestModel from "../../src/generated/src/models";
```

Comment:
This test cannot be public, since it's importing RestModel from generated which is internal and not publicly exposed

Reply:
My bad, I thought that generated ones are public by default.

Reply:
this is actually probably why the live tests are failing